src/PubSub.ts

```diff
@@ -4,7 +4,7 @@ import { CustomAsyncIterator } from './CustomAsyncIterator';
 
 type Listener = (...args: any[]) => any;
 export type Unsubscribe = () => any | boolean;
-export type Handler = (broadcast: Function, options?: Object) => Unsubscribe;
+export type Handler<T = any> = (broadcast: Function, options?: T) => Unsubscribe;
```

Comment:
Could you please use `unknown` instead of `any` here.

Reply:
As background: when we make a new release I would like to adjust the types accordingly so that we have more type safety (I'll be happy to adjust the other places.).